frontend/src/pages/org/workflow-detail.ts

```diff
@@ -497,6 +503,19 @@ export class WorkflowDetail extends LiteElement {
           <span> ${msg("Edit Crawler Instances")} </span>
         </sl-button>`;
     }
+    if (this.activePanel === "logs") {
+      const authToken = this.authState!.headers.Authorization.split(" ")[1];
+      return html` <h3>${this.tabLabels[this.activePanel]}</h3>
+        <sl-button
+          href=${`/api/orgs/${this.orgId}/crawls/${this.lastCrawlId}/logs?auth_bearer=${authToken}`}
+          download=${`btrix-${this.lastCrawlId}-logs.txt`}
+          size="small"
+          ?disabled=${!this.logs?.total}
+        >
+          <sl-icon slot="prefix" name="download"></sl-icon>
+          ${msg("Download Logs")}
```

Comment:
Doesn't have to be on the button necessarily, could display the crawl id or a link to the crawl at the top, something like that?

Reply:
Is the `/logs` only available for finished crawls? I assumed you could download logs for the currently running logs. If not, we could also disable this button so it's not confusing as to which crawl logs you're downloading, since you can always go to the last finished crawl's error logs tab to download them.

Reply:
Went ahead with disabling the button in the meantime:
<img width="330" alt="Screenshot 2023-10-02 at 3 42 44 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/7687a94f-cb38-4a20-9213-a32591ccd427">